README.md

```diff
@@ -1,63 +1,136 @@
-# sonu - Capstone-project
 
-- Responsive site from Mobile to desktop.
-- Based on [Cindy Shin in Behance](https://www.behance.net/adagio07) design.
-- Rock festival thematic.
-- Artist images from:
-  
-- Location image:
- 
-- Icons source: [Google icons](https://fonts.google.com/icons)
-- Png images: [PNG Wing](https://www.pngwing.com/)
 
-## Video presentation [link](https://www.loom.com/)
+  <h3><b>Sony Music, a WebSite about singers  and Events for singers</b></h3>
 
-## Built With:
 
-- HTML.
-- CSS.
-- JavaScript
-- Check the code with Linters tests:
-  _Webhint_
-  _Stylelint_
-  _Lighthouse_
-  _esLint_
 
-## Live Demo
+# 📗 Table of Contents
 
-[Live Demo]()
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-## Getting Started
+# 📖 [Sony Music] <a name="about-project"></a>
 
-To get a local copy up and running follow these simple example steps:
+**[Sony Music]** is a WebSite Project where you can find information about the singing event series.
 
-- Go to the folder where you want to store the project, and clone the repository:
-- SSH: git@github.com:zkr024/Capstone-Project.git
+## 🛠 Built With <a name="built-with"></a>
 
-## Authors
+### Tech Stack <a name="tech-stack"></a>
 
-👤 sonu prajapati 
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">Javascript</a></li>
+  </ul>
 
-- GitHub: [@githubhandle](https://github.com/sonuengineer)
-- Twitter: [@twitterhandle](https://twitter.com/)
-- LinkedIn: [LinkedIn](www.linkedin.com/in/)
 
-## 🤝 Contributing
+### Key Features <a name="key-features"></a>
 
-Contributions, issues, and feature requests are welcome!
+- **CSS rules are used to show/hide the links in the main NavBar according to the page you're on.**
+- **JavaScript routine is used to load the cards listed on the Featured Singers section.**
+- **You can see it on Mobile and Desktop**
+- **At the home page, in mobile version, you can expand the list of Featured Singers or hide them as well using the More/Less button.**
+- **In mobile, you can access the menu pressing the Burger icon, you can close the menu using the X in top/right corner, or chosing an option from the menu.**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
Great job @sonuengineer 🤝

- [ ] As stated in the project requirements, kindly deploy your project with the help of either GitHub pages or Netilify and add a link to the readme file.